packages/core/src/components/SessionTimeout/SessionTimeout.stories.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
```

Comment:
can you share the story url?

Reply:
https://rawgit.com/ripcurld/patternfly-react/sessiontimeout-storybook/index.html